### PR TITLE
AI in help center: include warning about AI being sometimes inaccurate

### PR DESCRIPTION
**Pull request by artpi**
This adds a warning that AI-generated content may produce inaccurate results
Zrzut ekranu 2024-01-4 o 17 44 20

## Proposed Changes

* Include the text

## Testing Instructions

1. Fire up this branch
2. Open Help center  - (?) in top right
3. Chose Email
4. Ask a question
5. Wait for AI generated answer
6. Scroll to bottom
7. Marvel at the new thought-out and accurate depiction of the AI capabilities

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/help-center/src/components/help-center-gpt.tsx [hunk 1]
```diff
@@ -197,7 +197,10 @@ export function HelpCenterGPT( { onResponseReceived }: Props ) {
 									) }
 								</div>
 								<GPTResponseDisclaimer>
-									{ __( "Generated by WordPress.com's Support AI", __i18n_text_domain__ ) }
+									{ __(
+										"Generated by WordPress.com's Support AI. Please be advised, that AI-generated content can be inaccurate or biased.",
```
Comment (Aurorum) [inline packages/help-center/src/components/help-center-gpt.tsx]: Typo?

```suggestion
										"Generated by WordPress.com's Support AI. Please be advised that AI-generated content can be inaccurate or biased.",
```

Comment (artpi) [reply packages/help-center/src/components/help-center-gpt.tsx]: I already changed this text, yeah it was a typo

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/10671180

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @artpi for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/10671180).

**Pull request merged**